scripts/install

```diff
@@ -158,61 +162,13 @@ cp --archive ../conf/plugins.local.php            $final_path/conf
 cp --archive ../conf/plugins.local.php            $final_path/conf/plugins.local.php.bak
 
 #=================================================
-# STORE THE CHECKSUM OF THE CONFIG FILE
-#=================================================
-
-# Calculate and store the config file checksum into the app settings
-#ynh_store_file_checksum --file="$final_path/conf/local.protected.php"
-### Files '$final_path/conf/local.php' and '$final_path/conf/acl.auth.php' can be modified by user, no need to store checksum as they cannot be overwritten safely by the upgrade script
-
-# #=================================================
-# # GENERIC FINALIZATION
-# #=================================================
-# # SECURE FILES AND DIRECTORIES
-# #=================================================
-
-# # Try to use "least privilege" to grant minimal access
-# # For details, see https://www.dokuwiki.org/install:permissions
-
-# # Files owned by DokuWiki can just read
-# chown -R root: $final_path
-
-# # DokuWiki needs to write inside these folders. Make "DokuWiki" owner
-# chown $app:root $final_path/{conf,inc}
-
-# # Make "DokuWiki" owner of configuration files that must be writable
-# chown $app:root $final_path/conf/{local.php,local.php.bak,users.auth.php,acl.auth.php,plugins.local.php,plugins.local.php.bak}
-
-# # Usefull for some plugins like https://www.dokuwiki.org/plugin:siteexport
-# # See https://www.dokuwiki.org/devel:preload
-# chown $app:root $final_path/inc/preload.php
-
-# # Grant read-only to all files as files copied above are owned by root by defaut and nginx cannot read them
-# # There are only files in the folder and there are no sublevels. No need to use "find"
-# chmod -R a+r $final_path/{conf,inc}
-
-# # Give write access to "data" and subfolders
-# chown -R $app:root $final_path/data
-# # Remove access to "other"
-# chmod -R o-rwx $final_path/data
-
-# # Allow the web admin panel to run, aka "Extension Manager"
-# chown -R $app:root $final_path/lib/plugins
-# # Allow to install templates
-# chown -R $app:root $final_path/lib/tpl
-
-# # Allow access to public assets like style sheets
-# find $final_path/lib -type f -print0 | xargs -0 chmod 0644
-# find $final_path/lib -type d -print0 | xargs -0 chmod 0755
-# # Using "find" instead of "chmod -R 755" so files does not become executable too
-# # chmod :   -rwxr-xr-x  1 root root  241 May  3 08:36 index.html   => BAD
-# # find  :   -rw-r--r--  1 1001 1002  241 May  3 08:36 index.html   => GOOD
-
+# GENERIC FINALIZATION
 #=================================================
 # SETUP FAIL2BAN
 #=================================================
 ynh_script_progression --message="Configuring Fail2Ban..." --weight=7
 
+# Create a dedicated Fail2Ban config
 ynh_add_fail2ban_config --logpath="/var/log/nginx/$domain-error.log" --failregex="^.*authentication failure. while reading response header from upstream, client: <HOST>,.*POST $path_url.*$" --max_retry=5
```

Comment:
### questions about fail2ban

I removed support for fail2ban because plugins which implement it is not available anymore

1. Is fail2ban really used by other yunohost apps?
3. is it useful for protection of apps?
4. does YunoHost protects users against bruteforce ldap login already? (my account will be disabled for X minutes if a attacker try to bruteforce?) So, "no need" for fail2ban in the application itself

I asked some help regarding this on the [dokuwiki forum](https://forum.dokuwiki.org/d/20414-protection-against-bruteforce-login) and got some feedback but I'm not sure if implementing some of this is "not against users" (they asked nothing regarding this) so I don't know ^^

Reply:
1. it seems that 47 apps use fail2ban: https://github.com/search?l=&q=org%3AYunoHost-Apps+fail2ban+filename%3Ainstall&type=code
2. yes it's useful, it will ban IP at the server level if someone try to brute force the app
3. as far as what i see in the code, fail2ban is not implemented for ldap https://github.com/YunoHost/yunohost/blob/dev/conf/fail2ban/yunohost-jails.conf

Reply:
Thanks for your detailed answer. I will look to add back fail2ban then.

About integrating it in the "core of yunohost", it should be done inside `slapd` with `ppolicy`where you can configure something like "block user account after X bad try)
See:
- https://sergio.24eme.fr/2017/04/27/mise-en-place-politique-mot-de-passe-openldap/
- https://www.zytrax.com/books/ldap/ch6/ppolicy.html

I don't know enough about yunohost user management to tell if it can be implemented "easily"